### PR TITLE
Updating the website whitelist

### PR DESCRIPTION
**Pull request by ojrb**
This  PR solved the issue #3990 , I have added the website **whiteboard.microsoft.com** to the whitelist section on the document, please review and merge !

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: devices/surface-hub/whiteboard-collaboration.md [hunk 1]
```diff
@@ -66,4 +66,4 @@ After you’re done, you can export a copy of the Whiteboard collaboration for y
 ## Related topics
 
 - [Windows 10 Creators Update for Surface Hub](https://www.microsoft.com/surface/support/surface-hub/windows-10-creators-update-surface-hub)
-- [Support documentation for Microsoft Whiteboard](https://support.office.com/en-us/article/Whiteboard-Help-0c0f2aa0-b1bb-491c-b814-fd22de4d7c01)
\ No newline at end of file
+- [Support documentation for Microsoft Whiteboard](https://support.office.com/en-us/article/Whiteboard-Help-0c0f2aa0-b1bb-491c-b814-fd22de4d7c01)
```
Comment (nenonix) [inline devices/surface-hub/whiteboard-collaboration.md]:
```suggestion
- [Support documentation for Microsoft Whiteboard](https://support.office.com/article/Whiteboard-Help-0c0f2aa0-b1bb-491c-b814-fd22de4d7c01)
```

File: windows/security/threat-protection/auditing/apply-a-basic-audit-policy-on-a-file-or-folder.md [hunk 2]
```diff
@@ -36,6 +36,21 @@ To complete this procedure, you must be logged on as a member of the built-in Ad
     -   To audit successful events, click **Success.**
     -   To audit failure events, click **Fail.**
     -   To audit all events, click **All.**
+6.  In the **Applies to** box, click the object or objects that the audit of events will apply to. It can be to:
```
Comment (nenonix) [inline windows/security/threat-protection/auditing/apply-a-basic-audit-policy-on-a-file-or-folder.md]:
```suggestion
6.  In the **Applies to** box, click the object or objects that the audit of events will apply to. These include:
```

File: windows/security/threat-protection/auditing/apply-a-basic-audit-policy-on-a-file-or-folder.md [hunk 3]
```diff
@@ -36,6 +36,21 @@ To complete this procedure, you must be logged on as a member of the built-in Ad
...
+    -   **This folder only.**
```
Comment (nenonix) [inline windows/security/threat-protection/auditing/apply-a-basic-audit-policy-on-a-file-or-folder.md]:
```suggestion
    -   **This folder only**
```

File: windows/security/threat-protection/auditing/apply-a-basic-audit-policy-on-a-file-or-folder.md [hunk 4]
```diff
@@ -36,6 +36,21 @@ To complete this procedure, you must be logged on as a member of the built-in Ad
...
+    -   **This folder, subfolders and files.**
```
Comment (nenonix) [inline windows/security/threat-protection/auditing/apply-a-basic-audit-policy-on-a-file-or-folder.md]:
```suggestion
    -   **This folder, subfolders and files**
```

File: windows/security/threat-protection/auditing/apply-a-basic-audit-policy-on-a-file-or-folder.md [hunk 5]
```diff
@@ -36,6 +36,21 @@ To complete this procedure, you must be logged on as a member of the built-in Ad
...
+    -   **This folder and subfolders.**
```
Comment (nenonix) [inline windows/security/threat-protection/auditing/apply-a-basic-audit-policy-on-a-file-or-folder.md]:
```suggestion
    -   **This folder and subfolders**
```

File: windows/security/threat-protection/auditing/apply-a-basic-audit-policy-on-a-file-or-folder.md [hunk 6]
```diff
@@ -36,6 +36,21 @@ To complete this procedure, you must be logged on as a member of the built-in Ad
...
+    -   **This folder and files.**
```
Comment (nenonix) [inline windows/security/threat-protection/auditing/apply-a-basic-audit-policy-on-a-file-or-folder.md]:
```suggestion
    -   **This folder and files**
```

File: windows/security/threat-protection/auditing/apply-a-basic-audit-policy-on-a-file-or-folder.md [hunk 7]
```diff
@@ -36,6 +36,21 @@ To complete this procedure, you must be logged on as a member of the built-in Ad
...
+    -   **Subfolders and files only.**
```
Comment (nenonix) [inline windows/security/threat-protection/auditing/apply-a-basic-audit-policy-on-a-file-or-folder.md]:
```suggestion
    -   **Subfolders and files only**
```

File: windows/security/threat-protection/auditing/apply-a-basic-audit-policy-on-a-file-or-folder.md [hunk 8]
```diff
@@ -36,6 +36,21 @@ To complete this procedure, you must be logged on as a member of the built-in Ad
...
+    -   **Subfolders only.**
```
Comment (nenonix) [inline windows/security/threat-protection/auditing/apply-a-basic-audit-policy-on-a-file-or-folder.md]:
```suggestion
    -   **Subfolders only**
```

File: windows/security/threat-protection/auditing/apply-a-basic-audit-policy-on-a-file-or-folder.md [hunk 9]
```diff
@@ -36,6 +36,21 @@ To complete this procedure, you must be logged on as a member of the built-in Ad
...
+    -   **Files only.**
```
Comment (nenonix) [inline windows/security/threat-protection/auditing/apply-a-basic-audit-policy-on-a-file-or-folder.md]:
```suggestion
    -   **Files only**
```

File: windows/security/threat-protection/auditing/apply-a-basic-audit-policy-on-a-file-or-folder.md [hunk 10]
```diff
@@ -36,6 +36,21 @@ To complete this procedure, you must be logged on as a member of the built-in Ad
...
+7.  By default, the selected **Basic Permissions** to Audit are the following:
```
Comment (nenonix) [inline windows/security/threat-protection/auditing/apply-a-basic-audit-policy-on-a-file-or-folder.md]:
```suggestion
7.  By default, the selected **Basic Permissions** to audit are the following:
```

File: windows/security/threat-protection/auditing/apply-a-basic-audit-policy-on-a-file-or-folder.md [hunk 11]
```diff
@@ -36,6 +36,21 @@ To complete this procedure, you must be logged on as a member of the built-in Ad
...
+    -   **Read & Execute.**
```
Comment (nenonix) [inline windows/security/threat-protection/auditing/apply-a-basic-audit-policy-on-a-file-or-folder.md]:
```suggestion
    -   **Read and execute**
```

File: windows/security/threat-protection/auditing/apply-a-basic-audit-policy-on-a-file-or-folder.md [hunk 12]
```diff
@@ -36,6 +36,21 @@ To complete this procedure, you must be logged on as a member of the built-in Ad
...
+    -   **List folder contents.**
```
Comment (nenonix) [inline windows/security/threat-protection/auditing/apply-a-basic-audit-policy-on-a-file-or-folder.md]:
```suggestion
    -   **List folder contents**
```

File: windows/security/threat-protection/auditing/apply-a-basic-audit-policy-on-a-file-or-folder.md [hunk 13]
```diff
@@ -36,6 +36,21 @@ To complete this procedure, you must be logged on as a member of the built-in Ad
...
+    -   **Read.**
```
Comment (nenonix) [inline windows/security/threat-protection/auditing/apply-a-basic-audit-policy-on-a-file-or-folder.md]:
```suggestion
    -   **Read**
```

**Comment by e0i** [comment]
@levinec @Mir0sh The PR that solves issue #3990 that has passed copy review is available for technical review. Thanks.

**Comment by e0i** [comment]
@levinec @Mir0sh Following up for updates. Thanks.

**Pull request merged**